CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+## CHANGELOG
+
+### 1.2.0 (2020-07-21)
+**SDK**
+- Upgraded build setup and some internal dependencies
+- Added support for customization of user agent information
+- Added feature to support backward & forward navigation support in a mini app
```

Comment:
I think, we could write: `Added backward & forward navigation support in a mini app`

Reply:
I think it is fine either way.

Reply:
No, there is a difference in both statements.